openfisca_core/json_to_test_case.py

```diff
@@ -1,85 +1,100 @@
 # -*- coding: utf-8 -*-
 
-import itertools
-
 from . import conv
+from copy import deepcopy
```

Comment:
Separate standard library imports from local imports (like it was before your commit with `itertools`)